src/app/features/location/edit-location/edit-location.component.html

```
@@ -1,4 +1,8 @@
-<div class="flex-row gap-regular align-end">
+<div
```

HTMLHint (1.9.2)

[error] Doctype must be declared before any non-comment content.

(doctype-first)